### DISCUSSION TITLE
Online beatmap (and possibly skin) sync

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#27376] [Ideas]
I think it would be great if you could sync all of your beatmaps and skins, so that you can redownload everything if you delete your game. Kinda like Steam where your games and their configs are stored on the cloud, so no matter what you can reinstall your games on **any device that supports Steam** (?).

**Comment by Gabixel** [comment]
Possibly related discussions/feedback that I was able to find:
- https://github.com/ppy/osu/discussions/22073:
  > Cloud Save maps, collections and local scores - osu!supporter?
- https://github.com/ppy/osu/discussions/26737
- https://github.com/ppy/osu/discussions/19477
- https://github.com/ppy/osu/discussions/23736 - not strictly related, but it's still about storing skins online

I don't know if this will ever happen, but it's certainly of very low priority right now

**Reply by StalkyBoi43815** [reply]
> Possibly related discussions/feedback that I was able to find:
> - https://github.com/ppy/osu/discussions/22073:
>   > Cloud Save maps, collections and local scores - osu!supporter?
> - https://github.com/ppy/osu/discussions/26737
> - https://github.com/ppy/osu/discussions/19477
> - https://github.com/ppy/osu/discussions/23736 - not strictly related, but it's still about storing skins online
> 
> I don't know if this will ever happen, but it's certainly of very low priority right now

Sadge, thanks for extensive answer anyways.

**Comment by first-storm** [comment]
Online skin sync might consume a lot of server storage. However, I believe that online config and playlist sync are possible.
